pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -122,6 +122,11 @@ func NewConfigObserver(
 				openShiftOnlyFeatureGates,
 				[]string{"extendedArguments", "feature-gates"},
 			),
+			featuregates.NewObserveFeatureFlagsFunc(
```

Comment:
Will this be pushing OpenShift-specific feature-gates to KCM? How's it going to influence the feature gates pushed from the above feature flags observer?

I think we need at least a comment explaining why there are two feature flags observers

Reply:
Comment added and content updated.